daemon/algod/api/server/v2/test/handlers_resources_test.go

```diff
@@ -107,6 +108,33 @@ func (l *mockLedger) LookupAsset(rnd basics.Round, addr basics.Address, aidx bas
 	}
 	return ar, nil
 }
+
+func (l *mockLedger) LookupAssets(rnd basics.Round, addr basics.Address, assetIDGT basics.AssetIndex, limit uint64) ([]ledgercore.AssetResourceWithIDs, basics.Round, error) {
+	ad, ok := l.accounts[addr]
+	if !ok {
+		return nil, rnd, nil
+	}
+
+	var res []ledgercore.AssetResourceWithIDs
+	for i := assetIDGT + 1; i < assetIDGT+1+basics.AssetIndex(limit); i++ {
```

Comment:
This particular test is contrived this way for my sanity - since the underlying db queries are not being invoked (and are tested elsewhere), I didn't see this as a problem.